apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+  openapi: 3.0.0
+  info:
+    title: Experiment API(s)
+    description: "- The Experiment API is used to create an experiment with user or device specifications. The Operations include create, get experiment  \n- The URL for Experiment API(S) is\
```

Comment:
The Experiment API is used to run experiments on the efficacy of new features for a set of targeted users using user and device specifications. The API supports operations to create experiments and retrieve an already existing experiments. \n- The URL for Experiment API(S) is\
      \ `/data/v1/experiment